test/runner/run_workers_test.js

```diff
@@ -111,4 +110,13 @@ describe('CodeceptJS Workers Runner', function () {
       done();
     });
   });
+
+  it('should create output folder with custom name', function (done) {
+    if (!semver.satisfies(process.version, '>=11.7.0')) this.skip('not for node version');
+    exec(`${codecept_run_glob('codecept.workers-custom-output-folder-name.conf.js')} 1 --grep "grep" --debug`, (err, stdout, stderr) => {
+      stdout.should.include('customOutput_'); // feature
```

Comment:
As said in #2154 issue, when folder does not creates, error appears:
`ENOENT: no such file or directory, open '/Users/WebstormProjects/e2e/.codeceptjs-output/test.png'`

And it looks like, stdout has folder name always. No matter successful test run or not.

Did you check, that without your changes in `run-workers.js` test fails?

Reply:
Oh you check, that there are no `err`. Then ok, sorry

Reply:
Nevertheless, test name says codecept `should create output folder`, but it checks for stdout only, and no folder existing

Reply:
update the test to verify if the folder is created.